namespaces/live.cloud-platform.service.justice.gov.uk/data-platform-app-cica-information-extraction-tool-dev/00-namespace.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: "data-platform-app-cica-information-extraction-tool-dev"
+  labels:
+    cloud-platform.justice.gov.uk/is-production: "false"
+    cloud-platform.justice.gov.uk/environment-name: "development"
+    pod-security.kubernetes.io/enforce: restricted
+  annotations:
+    cloud-platform.justice.gov.uk/business-unit: "HQ"
+    cloud-platform.justice.gov.uk/slack-channel: "cica-data-science-"
```

Comment:
Is this a valid slack channel?

Reply:
Hi Mike, yeah, that's the correct channel name. Not sure why it was set-up with a dash at the end